.gitignore

```diff
@@ -0,0 +1,4 @@
+*.iml
+.idea/
```

Comment:
I was lazy and allowed my IDE to automatically generate this :)
Both are already in my global ignore settings